docs/zkapps/o1js/bitwise-operations.mdx

```diff
@@ -0,0 +1,303 @@
+---
+title: Bitwise Gadgets
+hide_title: true
+description: Bitwise operations in o1js are implemented as gadgets.
+keywords:
+  - gadgets
+  - bitwise operations
+  - o1js
+---
+
+:::info
+
+Please note that zkApp programmability is not yet available on Mina Mainnet, but zkApps can now be deployed to Berkeley Testnet.
+
+:::
+
+# Bitwise Operations
+
+In o1js, bitwise operations are implemented as [gadgets](/zkapps/o1js/gadgets).
+
+- [and()](#and)
+- [not()](#not)
+- [xor()](#xor)
+- [leftShift()](#leftshift)
+- [rightShift()](#rightshift)
+- [rotate()](#rotate)
+- [rangeCheck64()](#rangecheck64)
+- [multiRangeCheck()](#multirangecheck)
+- [compactMultiRangeCheck()](#compactmultirangecheck)
+
+See the type declaration for [Gadgets](/zkapps/o1js-reference/modules#gadgets) in the o1js Reference documentation.
+
+## and()
+
+```ts
```

Comment:
I grabbed this "syntax" from http://localhost:3000/zkapps/o1js-reference/modules#gadgets but it's really not a true command syntax, more of an explainer

Did we want to provide literal syntax here? or omit this for each gadget?